index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body class="layout">
+  <header class="header">
+    <nav>
+      <img src="images/burgers.svg" alt="Menu">
+    </nav>
+  </header>
+  <main>
+      <div class="headline">
```

Comment:
- [ ] Kindly consider replacing this div with `section` tag as it will be semantically correct and will represent a standalone generic section for the document.